src/common/Option/Option.spec.js

```diff
@@ -72,5 +72,5 @@ describe('Option', () => {
       .childAt(0)
       .matchesElement(element);
   const title = () => component.find('h5').text();
-  const description = () => component.find('p').text();
+  const content = () => component.find('p').html();
```

Comment:
Maybe that will allow us to test it as a node instead of string?
```suggestion
  const content = () => component.find('p').childAt(0);
```

Reply:
nice idea, but not, getting this error:

```javascript
Error: expect(received).toEqual(expected) // deep equality

- Expected
+ Received

- <p>
-   A content
- </p>
+ ShallowWrapper {}
```